show/fgnhg.py

```diff
@@ -113,7 +113,7 @@ def hash_view(nhg):
         for nhg_prefix in table_keys:
             bank_dict = {}
             t_dict = state_db.get_all(state_db.STATE_DB, nhg_prefix)
-            vals = sorted(set([val for val in list(t_dict.values())]))
+            vals = sorted(set([val for val in t_dict.values()]))
```

Comment:
Similar to above one

Reply:
Addressed in latest commit.

Reply:
FYI, I needed to change this back, because the conversion to a set eliminates duplicate values. Unit tests were failing.